### ISSUE TITLE
Release v1.31.5

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#1167]
:robot: A dedicated issue to run tests and deploy release [v1.31.5](https://github.com/deckhouse/deckhouse/milestone/32).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | gcp | azure | openstack | yandex-cloud | vsphere | static`
  - `git_ref` is a tag or branch: `vX.Y.Z | test-X.Y.Z* | main | release-X.Y`
- `/e2e/use/cri/<cri_name>` specifies which CRI to use for e2e test.
  - `cri_name` is one of `containerd | docker`
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.19 | 1.20 | 1.21 | 1.22`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag or branch: `vX.Y.Z | test-vX.Y.Z* | main | release-X.Y`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.21'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws
/e2e/use/cri/docker
/e2e/use/cri/containerd
/e2e/use/k8s/1.19
/e2e/use/k8s/1.21

This comment will run 4 e2e jobs on AWS with Docker and containerd
and with Kubernetes version 1.19 and 1.21.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build modules FE' and 'build FE' jobs are finished.

</details>

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @diafour. I've recognized your '/build' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
New tag **v1.31.5** is created.

**Comment by deckhouse-BOaTswain** [comment]
New commit [1c7568](https://github.com/deckhouse/deckhouse/commit/1c7568ee92984f64d40c07068aad687878c3615a) in branch **main**:
```
[docs] updated prices for managed k8s (#1171)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [3c5f4c](https://github.com/deckhouse/deckhouse/commit/3c5f4cc828a2449190f38242c260741e55761467) in branch **main**:
```
[docs] Minor fixes (#1172)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [3f9f5f](https://github.com/deckhouse/deckhouse/commit/3f9f5f1eb97f7ee44b2c2700f7a772a329b86060) in branch **main**:
```
Re-generate changelog (#1175)

Co-authored-by: z9r5 <z9r5@users.noreply.github.com>
```
